shell/browser/auto_updater_mac.mm

```diff
@@ -86,9 +86,6 @@
         forHTTPHeaderField:base::SysUTF8ToNSString(it.first)];
   }
 
-  if (g_updater)
```

Comment:
https://github.com/electron/electron/blob/d02c9f8bc644ba39acf9caa908c33a737202a790/shell/browser/auto_updater_mac.mm#L28

this is an obj-c pointers so we 1) should add a strong ref to it and 2) perhaps explicitly set it to `nil` here instead?

Reply:
Ah perfect, this makes sense - added strong to the definition on like 28 and set to to nil if it exists, thanks!